### ISSUE TITLE
Default upgrade strategy for Control plane is should be 1 when provisioning RKE2 clusters

### ISSUE THREAD

**Issue by sowmyav27** [rancher/dashboard#5112]
**on 2.6-head commit `11a7451`**

To be in Parity with RKE1, the upgrade strategy of control plane nodes should be `1`

Currently the default value is 10%
<img width="1453" alt="Screen Shot 2022-02-13 at 3 38 48 PM" src="https://user-images.githubusercontent.com/26032343/153780509-29fa80d4-fb6f-45a0-bcb6-75d04f3d9370.png">

**Comment by nwmac** [comment]
@sowmyav27 So you're saying it should be 100% for Control Plane Concurrency? Can you include a screenshot of how this looks for RKE1?

**Comment by catherineluse** [comment]
The `concurrency` field indicates how many nodes can be upgraded at the same time, and that number is also the maximum number of nodes that are permitted to be unavailable during an upgrade. If the number of unavailable nodes is larger than the concurrency, the upgrade will fail.

For RKE1, by default, the minimum number of unavailable control plane nodes is 10 percent of control plane nodes, rounded down to the nearest node, with a minimum batch size of one node. The same goes for worker nodes. More details here: https://rancher.com/docs/rke/latest/en/upgrades/maintaining-availability/

So if it says 1, I would read that as meaning it will only upgrade one control plane node at a time.

We should probably explain how that works in plain English in the UI though, because I don't think that's very clear.

**Comment by sowmyav27** [comment]
This is for rke2 parity with rke1 clusters.
For rke1 clusters, this is how it is
<img width="1456" alt="Screen Shot 2022-03-29 at 7 44 28 PM" src="https://user-images.githubusercontent.com/26032343/160631683-b10e1d59-f2d5-4ee0-9312-4e5c6930682d.png">

**Comment by gaktive** [comment]
This should be an easy 1 or 2 line fix to set both Control Plane and Worker Node Concurrency to `1`.

Later on, we should review the UX here since the tooltip for mixed values seems very unclear.

**Comment by timhaneunsoo** [comment]
## Test Environment: ###
Rancher version: v2.6-head
Rancher cluster type: HA
Docker version: 20.10

Downstream cluster type: RKE2 node driver

----------

### Testing: ###

Tested this issue with the following steps:
1. Create RKE2 node driver cluster
2. Go to `Upgrade Strategy` under `Cluster Configuration` and check default values

**Result**

The default value is now 1
![image.png](https://zube.io/files/rancher/e5a8d1a63dd0ae028e875e1cf46fba89-image.png)

**Comment by Josh-Diamond** [comment]
It has come to my attention, that the desired default upgrade strategy for RKE2 worker-nodes should remain consistent with the worker-node defaults for RKE1.  This means changing the defaults for RKE2 worker-nodes back to 10%. Re-opening issue

**Comment by sowmyav27** [comment]
**On 2.6-head commit id: `311f462`**

- On UI
<img width="1433" alt="Screen Shot 2022-04-29 at 11 13 38 AM" src="https://user-images.githubusercontent.com/26032343/166001090-d3b10e0f-0733-4054-b866-93ac21ada2ae.png">

- UI sends
```
controlPlaneConcurrency: "1"
controlPlaneDrainOptions: {deleteEmptyDirData: true, disableEviction: false, enabled: true, force: false, gracePeriod: -1,…}
workerConcurrency: "10%"
```
